src/SQLProvider/Utils.fs

```diff
@@ -384,8 +431,8 @@ module Sql =
         }
 
     let executeSql createCommand sql (con:IDbConnection) =        
-        use com : IDbCommand = createCommand sql con   
-        com.ExecuteReader()    
+        let com : IDbCommand = createCommand sql con   
```

Comment:
Utils.fs class is used also by other providers. But `executeSql` is not executed by any interfaces or any common code. So I think you can add your version of the function to be spesifically in the Providers.Firebird.fs and do the disposable-handling there, leaving the common code to the current version. E.g. there is custom `executeSql` in Providers.MsSqlServer.fs (I don't know why).

And in the Providers.firebird.fs, maybe you don't need the whole type, if you just create a holder to keep reference of the cmd as long as you want, e.g., something like:
```fsharp
//either mutable or not, I don't know the case exactly.
let mutable myCmd = Unchecked.defaultof<IDisposable>
//then when needed:
myCmd <- (com : IDbCommand)

//later:
if myCmd <> Unchecked.defaultof<IDisposable> then
      myCmd.Dispose()
```